Tasks/AzureAppServiceManage/task.json

```diff
@@ -181,6 +206,19 @@
             }
         },
         {
+            "target": "AppInsightsResourceGroupName",
+            "endpointId": "$(ConnectedServiceName)",
+            "dataSourceName": "AzureResourceGroups"
```

Comment:
Is there a way to filter Azure resource groups that contain App Insights resource? This data source lists all resource groups irrespective of whether App insights resource is present or not.

Reply:
No there is no direct REST API for checking this. It could be achieved in this way - Get all app insights resources and get unique resource group of all. But we don't have support to get unique value as of now.